### DISCUSSION TITLE
Focus stealing prevention feature

### DISCUSSION THREAD

**Discussion by GHNewbiee** [WayfireWM/wayfire#2206] [Q&A]
Q: Is "focus stealing prevention (of new opened windows)" feature supported. *Tia*

**Comment by soreau** [comment]
There is a focus-stealing-prevention plugin for keystrokes in wayfire-plugins-extra.

**Comment by GHNewbiee** [comment]
@soreau *Tfyr*

> ... for keystrokes ...

Sorry, but it is not very clear to me. My case is as follows:

>After booting, I open N apps by ***mice*** clicking one-by-one. While apps getting open, I choose to work with a specific one.

Will the current plugin work in the above case?

*Tia*
